Makefile

```diff
@@ -20,15 +20,21 @@ clean_deb_source :
 	rm -rf net-snmp-${DEBIANVER}/
 	-rm .*.built
 
-${DEBS} : .clearwater1.built
+net-snmp-5.7.2~dfsg/COPYING:
+	apt-get source net-snmp=${DEBIANVER}-${UBUNTUVER}
+
+${DEBS}: .all_built
 
 .%.built : %.patch
 	touch $@.tmp
-	apt-get source net-snmp=${DEBIANVER}-${UBUNTUVER}
 	patch -p1 < $<
+	mv $@.tmp $@
+
+.all_built: net-snmp-5.7.2~dfsg/COPYING .clearwater1.built .clearwater2.built
```

Comment:
This doesn't guarantee order (in particular, `.clearwater2.built` depends on `.clearwater1.built` since it doesn't re-apply `clearwater1.patch`).